spec/code_objects/constants_spec.rb

```diff
@@ -50,19 +50,30 @@
 
   describe :BUILTIN_EXCEPTIONS do
     it "includes all base exceptions" do
+      bad_names = []
       YARD::CodeObjects::BUILTIN_EXCEPTIONS.each do |name|
-        expect(eval(name)).to be <= Exception
+         next if name == 'ClosedQueueError' && RUBY_VERSION < '2.3'
```

Comment:
Instead of manually mapping out constants (this will be a headache to maintain), why not just ignore any constant that does not exist? A NameError rescue should suffice.